packages/shared/styles/components/SfHeroItem.scss

```diff
@@ -0,0 +1,82 @@
+@import '../variables';
+
+$item-height: 18rem;
```

Comment:
`$hero-item-height`...
also, maybe is better to put it inside `SfHero.scss`, then you can use `&__item { ... }` and variables prefixed by `$hero__item`, wdyt?